### PR TITLE
Client install

### PR DESCRIPTION
**Pull request by julibert**
This pull request fixes the linker error due to the installation policy change in Micro XRCE-DDS Client.
Take into account that Micro XRCE-DDS Client shall point to develop branch once https://github.com/eProsima/Micro-XRCE-DDS-Client/pull/128 is merged.

### CODE REVIEW

File: micro_ros_setup/config/host/generic/client_host_packages.repos [hunk 1]
```diff
@@ -2,11 +2,11 @@ repositories:
   eProsima/Micro-CDR:
     type: git
     url: https://github.com/eProsima/micro-CDR.git
-    version: v1.1.0
+    version: v1.1.2
   eProsima/Micro-XRCE-DDS-Client:
     type: git
     url: https://github.com/eProsima/Micro-XRCE-DDS-Client.git
-    version: v1.1.3
+    version: feature/prefix
```
Comment (pablogs9) [inline micro_ros_setup/config/host/generic/client_host_packages.repos]: feaure/prefix merged, should we use develop branch?

Comment (julianbermudez) [reply micro_ros_setup/config/host/generic/client_host_packages.repos]: I changed `Micro-XRCE-DDS-Client` to develop branch and `rmw-microxrcedds` to feature/rmw_service.

**Comment by pablogs9** [comment]
I think that this PR is outdated, what do you think @julibert?

**Comment by julibert** [comment]
Yes, it is outdated. I move the PRs over this branch to `feature/multiboard`.

**Pull request closed without merging**